libshaderc_spvc/src/spvcir_pass.cc

```diff
@@ -639,6 +862,7 @@ void SpvcIrPass::GenerateSpirvCrossIR(Instruction *inst) {
       break;
     }
 
+    case SpvOpFAdd:
```

Comment:
Is the case fall through here intentional? If so please add a comment mentioning that.